### ISSUE TITLE
Create credrankHelp command for CLI help dialogs.

### ISSUE THREAD

**Issue by topocount** [sourcecred/sourcecred#2750]
No `credrankHelp` function exists like it does for other commands. These help dialogs explain what the command does in plain english. See [go](https://github.com/sourcecred/sourcecred/blob/60828abd12bab3666a98d8c53fa2be762e25b85b/src/cli/go.js#L43-L59) or any other commands for reference.
_Originally posted by topocount in https://github.com/sourcecred/sourcecred/issues/2749#issuecomment-777759900_

## AC:
* `scdev help` output includes `credrank` as an option
* `scdev help credrank` gives a specific help message about the credrank command and its available flags

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200032326400385/1200038305067838) by [Unito](https://www.unito.io/learn-more)

**Comment by Gregory-Pereira** [comment]
I will take a look now

**Comment by Gregory-Pereira** [comment]
I pushed a commit to my fork but it needs some work before its ready for a pull request because I literally started with the codebase today, this is what I have so far:
```js

export const credrankHelp: Command = async (args, std) => {
  std.out(
    dedent`\
      usage: sourcecred credrank [options]

      options:
      -d  Outputs the diff of changes compared to the last saved graph.
      -s, --simulation  Skips writing changes to the graph and ledger json.

      Loads in the existing graph, creates a bonus graph of bonus minted cred for dependencies, merges them, and rewrites the credGraph.

      Under the hood, this runs 'prepareCredData' to gather the existing dependencies and credGraph, 'computeBonusMinting' for the minting of bonus cred, 'merge' to combine them, and 'credrank' to re-rank the new graph, all asynchronously. If the -s or --simulate argument is provided, it will not persist the updated graph, and if the -d flag is provided it will attempt to load in the generate the new Graph and compare it to the original.
      `.trimRight()
  );
  return 0;
};
```

I need to read more into the codebase to understand what happens if any of these functions fail, and in what order.
